content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,167 @@
+cycle and the vibrant support from our community.
+
+This release consists of 45 enhancements. Of those enhancements, 17 have graduated to Stable, 18 are
+entering Beta, and 10 have graduated to Alpha.
+
+
+## Release theme and logo
+<Logo image size is recommended to be no more than 2160px> 
+[Kat requested to hold review for this section until after the theme is revealed, and other 
+sections are reviewed.]
+
+
+## Improvements that graduated to stable in Kubernetes v1.30 {#graduations-to-stable}
+
+_This is a selection of some of the improvements that are now stable following the v1.30 release._
+
+### Robust VolumeManager reconstruction after kubelet restart ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+This is a volume manager refactoring that allows the kubelet to populate additional information about how existing volumes are mounted during the kubelet startup. In general, this makes volume cleanup after kubelet restart or machine reboot more robust.
+
+This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is stable, the feature gate is locked and cannot be disabled.
+
+### Prevent unauthorised volume mode conversion during volume restore ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+Blurb for 3141 goes here.
+
+### Pod Scheduling Readiness ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3521 goes here.
+
+### Min domains in PodTopologySpread ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3022 goes here.
+
+### Go workspaces for k/k ([SIG Architecture](https://github.com/kubernetes/community/tree/master/sig-architecture))
+
+Blurb for 4402 goes here.
+
+## Improvements that graduated to beta in Kubernetes v1.30 {#graduations-to-beta}
+
+_This is a selection of some of the improvements that are now beta following the v1.30 release._
+
+### Node log query ([SIG Windows](https://github.com/kubernetes/community/tree/master/sig-windows))
+
+To help with debugging issues on nodes, Kubernetes v1.27 introduced a feature that allows viewing logs of services running on the node. To use the feature, ensure that the NodeLogQuery feature gate is enabled for that node, and that the kubelet configuration options enableSystemLogHandler and enableSystemLogQuery are both set to true. 
+
+On Linux the assumption is that service logs are available via journald. On Windows the assumption is that service logs are available in the application log provider. On both operating systems, logs are also available by reading files within /var/log/.
+[To read more about the feature, please click here](https://kubernetes.io/docs/concepts/cluster-administration/system-logs/#log-query)
```

Comment:
Consider using site-relative link.

```suggestion
[To read more about the feature, please click here](/docs/concepts/cluster-administration/system-logs/#log-query)
```

Reply:
We also try to avoid “click here” (instead, use the section title as the anchor)

Reply:
> We also try to avoid “click here” (instead, use the section title as the anchor)

Would you please add an updated suggestion for the link / link text?

Reply:
@dipesh-rawat  @sftim  i have updated the title to incorporate both your changes.